<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actiontest"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuBackGround">
     <property name="title">
      <string>BackGround</string>
     </property>
     <addaction name="actionBlack"/>
     <addaction name="actionWhite"/>
     <addaction name="actionSelect"/>
    </widget>
    <addaction name="menuBackGround"/>
    <addaction name="actionCoordinate"/>
   </widget>
   <widget class="QMenu" name="menuTrajectory_Processing">
    <property name="title">
     <string>Trajectory Processing</string>
    </property>
    <addaction name="actionTarget_Center"/>
    <addaction name="actionInverse_Calculation_GCP"/>
    <addaction name="actionGCP_Import"/>
    <addaction name="actionGCP_Associate"/>
    <addaction name="actionPretreatment"/>
    <addaction name="actionTrajectory_Optimization"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTrajectory_Processing"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionBlack">
   <property name="text">
    <string>Black</string>
   </property>
  </action>
  <action name="actionWhite">
   <property name="text">
    <string>White</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="actionTarget_Cacu">
   <property name="text">
    <string>Target Cacu</string>
   </property>
  </action>
  <action name="actionInverse_Calculation_GCP">
   <property name="text">
    <string>Inverse Calculation GCP</string>
   </property>
  </action>
  <action name="actionTarget_Center">
   <property name="text">
    <string>Target Center</string>
   </property>
  </action>
  <action name="actionGCP_Associate">
   <property name="text">
    <string>GCP Associate</string>
   </property>
  </action>
  <action name="actionTrajectory_Optimization">
   <property name="text">
    <string>Trajectory Optimization</string>
   </property>
  </action>
  <action name="actionGCP_Import">
   <property name="text">
    <string>GCP Import</string>
   </property>
  </action>
  <action name="actionPretreatment">
   <property name="text">
    <string>Pretreatment</string>
   </property>
  </action>
  <action name="actionCoordinate">
   <property name="text">
    <string>Coordinate</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
